source/modules/.scaffold/scaffold.js

```diff
@@ -1,59 +1,60 @@
-/*!
+import EstaticoModule from '../../assets/js/helpers/module';
+
+const name = '{{name}}';
+
+/*
  * {{originalName}}
  *
  * @author
  * @copyright
  */
-import EstaticoModule from '../../assets/js/helpers/module';
-
-class {{className}} extends EstaticoModule {
+export default class {{className}} extends EstaticoModule {
+	static events = {
```

Comment:
Did not see any major problems there, most problems are caused by wrong usage - e.g. some guys did not understand that a class member belongs to a class and not to the instance and of course the inheritance works different.